<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget_1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>611</width>
      <height>211</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>300</number>
    </attribute>
    <column>
     <property name="text">
      <string>Doktorlar</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Alınan Toplam Randevu Sayısı</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_rgeri">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>390</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Geri</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
